libraries/AP_Baro/AP_Baro_MS5611.cpp

```diff
@@ -152,8 +196,11 @@ bool AP_Baro_MS56XX::_init()
     case BARO_MS5611:
         devtype = DEVTYPE_BARO_MS5611;
         break;
-    case BARO_MS5837:
-        devtype = DEVTYPE_BARO_MS5837;
+    case BARO_MS5837_30BA:
+        devtype = DEVTYPE_BARO_MS5837_30BA;
+        break;
+    case BARO_MS5837_02BA:
+        devtype = DEVTYPE_BARO_MS5837_02BA; // Reuse the same dev type for both variants
```

Comment:
```suggestion
        devtype = DEVTYPE_BARO_MS5837_02BA;
```
This comment doesn't make sense if the devtypes are independent.